backend/config/corpora-api.yml

```diff
@@ -391,6 +391,46 @@ paths:
         "404":
           $ref: "#/components/responses/404"
 
+  /dp/v1/datasets/{dataset_uuid}/gene_sets:
+    post:
+      tags:
+        - datasets
+        - genesets
+      summary: Update links between a dataset and gene sets
+      security:
+        - cxguserCookie: [ ]
+      description: >-
+        Request to update (add or remove) linkages between a dataset and gene sets. The request must be made by the
+        collection owner and the dataset and gene sets must all belong to the same private collection
+      operationId: corpora.lambdas.api.v1.dataset.post_dataset_gene_sets
+      parameters:
+        - $ref: "#/components/parameters/path_dataset_uuid"
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - add
```

Comment:
Looking at this definition it's not clear where `add` and `remove` come from.  It would make more sense to add this required section to */components/schemas/dataset_gene_set_linkage_update*.

Reply:
updated to remove dataset_gene_set_linkage_update with add/remove defined directly in the response